### PR TITLE
New tee type: CCA (Confidential Compute Architecture)

### PR DESCRIPTION
**Pull request by chendave**

### CODE REVIEW

File: src/api/Cargo.toml [hunk 1]
```diff
@@ -22,8 +22,8 @@ actix-web-httpauth = "0.8.0"
 aes-gcm = "0.10.1"
 anyhow.workspace = true
 async-trait.workspace = true
-as-types = { git = "https://github.com/confidential-containers/attestation-service.git", tag = "v0.7.0"}
-attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git", default-features = false, tag = "v0.7.0", optional = true}
+as-types = { git = "https://github.com/chendave/attestation-service.git", branch = "fvp"}
+attestation-service = { git = "https://github.com/chendave/attestation-service.git", branch = "fvp", default-features = false, optional = true}
```
Comment (chendave) [inline src/api/Cargo.toml:26]: those change should be updated once the change in attestation-service is in and tag is created.

**Comment by chendave** [comment]
/cc @Xynnn007 @jialez0 @sameo @fitzthum

This is ready for review now, thanks!

File: tools/client/Cargo.toml [hunk 2]
```diff
@@ -21,7 +21,7 @@ base64.workspace = true
 clap = { version = "4.0.29", features = ["derive"] }
 env_logger.workspace = true
 jwt-simple = "0.11.4"
-kbs_protocol = { git = "https://github.com/confidential-containers/guest-components" }
+kbs_protocol = { git = "https://github.com/chendave/guest-components.git", branch = "fvp" }
```
Comment (fitzthum) [inline tools/client/Cargo.toml:24]: Do we still need this?

Comment (chendave) [reply tools/client/Cargo.toml:24]: @fitzthum, we don't, just need that pending PR to be merged, then this I will be updated. :)

**Review by fitzthum — APPROVED** [review]
LGTM

**Pull request merged**